app/screens/navigation.ts

```diff
@@ -678,10 +678,17 @@ export async function bottomSheetWithTeamList({title, teams, dimensions, renderC
     const NO_TEAMS_HEIGHT = 392;
     const maxHeight = Math.round((dimensions.height * 0.9));
 
+    const ADDITIONAL_BOTTOM_MARGIN = 34;
```

Comment:
To me this looks like the bottom inset on an iphone, consider using that instead of a constant